### PR TITLE
Travis-ci integration

### PR DESCRIPTION
**Pull request by gianarb**
Add .travis.yml and badge in README.md

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - 5.5
+  - 5.6
+  - 7
+
+before_install:
+ - composer self-update
+ - composer install --prefer-source
+
+script:
+  - ./vendor/bin/phpunit
```
Comment (samsonasik) [inline .travis.yml]: I think : 
```
./vendor/bin/phpunit --coverage-text
```
can show what classes that need improvement for test.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+language: php
+
+php:
+  - 5.3.3
```
Comment (samsonasik) [inline .travis.yml]: should be 5.3.23 , or just use 5.3

Comment (gianarb) [inline .travis.yml]: composer.json is PHP >= 5.3.3 :)
5.3 is exclused

Comment (Ocramius) [inline .travis.yml]: FYI, I don't mind bumping to 5.4 directly, but I don't have full control here.

This should indeed be 5.3, not 5.3.3

**Comment by Ocramius** [comment]
Merging first - further improvements in separate PRs please :)

**Pull request merged**